pkg/interactor/data.go

```diff
@@ -0,0 +1,59 @@
+package interactor
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+type DataInteractor struct {
+	DataStore data.DataStore
+}
+
+func NewDataInteractor(dataStore data.DataStore) DataInteractor {
+	return DataInteractor{dataStore}
+}
```

Comment:
In order to avoid allocations, as a rule of thumb, [always return a pointer](https://github.com/golang/go/wiki/CodeReviewComments#receiver-type) for constructors :eyes: 

```suggestion
func NewDataInteractor(dataStore data.DataStore) *DataInteractor {
	return &DataInteractor{dataStore}
}
```

Reply:
Done!